src/style.css

```diff
@@ -1 +1,114 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+:root {
+  width: 100vw;
+  box-sizing: border-box;
+}
+
+.logo__link {
+  text-decoration: none;
+  height: 40px;
+}
+
+body {
+  margin: 0;
+}
+
+.nav__text {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  text-transform: uppercase;
+  font-size: 12px;
+  text-decoration: none;
+  list-style-type: none;
+}
+
+.nav__box {
+  display: flex;
+  height: 60px;
```

Comment:
do not harcode height use vertical paddings for links

Reply:
@SanyaBratashchuk  thank you! it was great idea with paddings for links
I overloaded code with 'height' parameter